translations/es.ts

```diff
@@ -1101,7 +1101,7 @@ en lugar de la barra de tareas del sistema.</translation>
     <message>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="200"/>
         <source>Your status is changed to Away after set period of inactivity.</source>
-        <translation>Tu estado se cambia a Ausente después del período de inactividad establecido.</translation>
+        <translation>Tu estado cambia a &apos;Ausente&apos; después del período de inactividad establecido.</translation>
```

Comment:
in 2340 (only a comment) quotation marks were used, even though it starts with a capital letter. Personally, I prefer statusses mentioned in a phraae or sentence as being makred by ' or ". 
But if you really dislike it, I will take them out. I will not insiste on them.